courses/fundamentals_of_ada/quiz/primitives_and_classwide/quiz.rst

```diff
@@ -0,0 +1,21 @@
+.. code:: Ada
+
+    type T1 is tagged null record;
+    type T2 is new T1 with null record;
+    -- Declarations
+    V : T2;
+
+Which of the following piece(s) of code allow for calling :ada:`Proc (V)`?
```

Comment:
Should we note that if these procedures are in a different scope, then they are fine?

Reply:
For answer C, right?

Reply:
Yes, but maybe better is a reminder that "primitive" only applies to subprograms in the same scope?